vhdbuilder/scripts/cni-prefetch-driver.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+CNI_PREFETCH_SCRIPT_PATH="/opt/azure/containers/cni-prefetch.sh"
```

Comment:
updated this path `/opt/azure/containers/prefetch.sh` after feedback in the other PR

Reply:
also, lets generalize towards `CONTAINER_IMAGE_PREFETCH_SCRIPT_PATH` as opposed to CNI-specific stuff - decided to go this route after discussion with @wedaly